docker_setup_scripts/ubuntu.sh

```diff
@@ -2,49 +2,48 @@
 
 set -euo pipefail
 
-echo "::group::apt update"
-apt-get update
-apt-get dist-upgrade -y
-echo "::endgroup::"
+# shellcheck source=docker_setup_scripts/docker_setup_scripts_common.sh
+. "${BASH_SOURCE%/*}/docker_setup_scripts_common.sh"
 
+# shellcheck disable=SC1091
 . /etc/lsb-release
 ubuntu_major_version=${DISTRIB_RELEASE%%.*}
 
 packages=(
-    apt-file
-    apt-utils
-    automake
-    bison
-    curl
-    flex
-    git
-    golang
-    groff-base
-    less
-    libasan5
-    libbz2-dev
-    libicu-dev
-    libncurses5-dev
-    libreadline-dev
-    libssl-dev
-    libtool
-    libtsan0
-    locales
-    maven
-    openjdk-8-jdk-headless
-    pkg-config
-    python3-dev
-    python3-pip
-    python3-venv
-    python3-wheel
-    rsync
-    sudo
-    tzdata
-    unzip
-    uuid-dev
-    vim
-    wget
-    xz-utils
+  apt-file
+  apt-utils
+  automake
+  bison
+  curl
+  flex
+  git
+  golang
+  groff-base
+  less
+  libasan5
+  libbz2-dev
+  libicu-dev
+  libncurses5-dev
+  libreadline-dev
+  libssl-dev
+  libtool
+  libtsan0
+  locales
+  maven
+  openjdk-8-jdk-headless
+  pkg-config
+  python3-dev
+  python3-pip
+  python3-venv
+  python3-wheel
+  rsync
+  sudo
+  tzdata
+  unzip
+  uuid-dev
+  vim
+  wget
+  xz-utils
 )
 
 if [[ $ubuntu_major_version -le 18 ]]; then
```

Comment:
This block could probably be removed too, our ubuntu docker base is 18.04

Reply:
If we remove this, we could also remove line 9 and 10 above.

Reply:
`-le` means less than or equal to. This is needed for Ubuntu 18.04.